### PR TITLE
[now-cli] Add renewal price to `now domains inspect`

### PR DESCRIPTION
**Pull request by mglagola**
Adds `Renewal Price` to `now domains inspect` command if the domain was bought with ZEIT.

### CODE REVIEW

File: packages/now-cli/src/commands/domains/inspect.ts [hunk 1]
```diff
@@ -104,32 +110,43 @@ export default async function inspect(
...
+    `    ${chalk.cyan('TXT Verified At')}\t\t${formatDate(
+      domain.txtVerifiedAt
+    )}\n`
   );
-  output.print(`    ${chalk.cyan('CDN Enabled')}\t\t${true}\n`);
+  renewalPrice &&
+    domain.boughtAt &&
+    output.print(
+      `    ${chalk.cyan('Renewal Price')}\t\t$${renewalPrice} USD\n`
+    );
```
Comment (styfle) [inline packages/now-cli/src/commands/domains/inspect.ts:133]: Nice! Please add a test for this new behavior 👍

File: packages/now-cli/src/commands/domains/inspect.ts [hunk 2]
```diff
@@ -104,32 +110,43 @@ export default async function inspect(
...
+    )}\n`
   );
   output.print(
-    `    ${chalk.cyan('TXT Verified At')}\t\t${formatDate(domain.txtVerifiedAt)}\n`
+    `    ${chalk.cyan('TXT Verified At')}\t\t${formatDate(
+      domain.txtVerifiedAt
+    )}\n`
   );
-  output.print(`    ${chalk.cyan('CDN Enabled')}\t\t${true}\n`);
+  renewalPrice &&
```
Comment (styfle) [inline packages/now-cli/src/commands/domains/inspect.ts]: Can you use a `if` statement here instead?

**Review by anatrajkovska — APPROVED** [review]

File: packages/now-cli/test/integration.js [hunk 3]
```diff
@@ -493,6 +493,35 @@ test('list the payment methods', async t => {
...
+
+  const rmRes = await execa(
+    binaryPath,
+    [`domains`, `rm`, domainName, ...defaultArgs],
+    { reject: false, input: 'y' }
+  );
+  t.is(rmRes.code, 0);
+
+  t.is(code, 0);
+  t.true(!stderr.includes(`Renewal Price`));
```
Comment (styfle) [inline packages/now-cli/test/integration.js:522]: Shouldn't this be looking at stdout to check if Renewal Price is found?

Comment (mglagola) [reply packages/now-cli/test/integration.js:522]: @styfle Renewal price is only for domains that were bought with us - the domain in this test was not (so we shouldn't include `Renewal Price` in the output)

Comment (styfle) [reply packages/now-cli/test/integration.js:522]: Let me rephrase my question: why are you using `stderr` instead of `stdout`?

Is that how `now domains inspect` is supposed to work? That seems wrong because it's not an error.

File: packages/now-cli/test/integration.js [hunk 4]
```diff
@@ -493,6 +493,35 @@ test('list the payment methods', async t => {
   t.true(stdout.startsWith(`> 0 cards found under ${contextName}`));
 });
 
+test('domains inspect', async t => {
+  const domainName = `inspect-${contextName}.org`;
```
Comment (styfle) [inline packages/now-cli/test/integration.js:497]: How did you choose this domain? 

I'm kinda worried about using .org because the [price cap was removed](https://arstechnica.com/tech-policy/2019/07/icann-eliminates-org-domain-price-caps-despite-lopsided-opposition/).

Is the test purchasing a new domain each time?

Comment (styfle) [reply packages/now-cli/test/integration.js:497]: Oh I see the `add` is for an existing domain. I see the other command is `now domains buy` 😅

**Review by styfle — APPROVED** [review]
Looks good, thanks!
